datasets/wit/README.md

```diff
@@ -0,0 +1,256 @@
+
+## Dataset Structure
+
+### Data Instances
+
+```
+{
+  'language': 'en', '
+  'page_url': 'https://en.wikipedia.org/wiki/Oxydactylus',
+  'image_url': 'https://upload.wikimedia.org/wikipedia/commons/5/5f/Oxydactylus_longipes_fm.jpg',
+  'page_title': 'Oxydactylus',
+  'section_title': None,
+  'hierarchical_section_title': 'Oxydactylus',
+  'caption_reference_description': None,
+  'caption_attribution_description': 'English: Mounted skeleton of Oxydactylus longipes in the Field Museum of Natural History.',
+  'caption_alt_text_description': None,
+  'mime_type': 'image/jpeg',
+  'original_height': 3564,
+  'original_width': 2748,
+  'is_main_image': True,
+  'attribution_passes_lang_id': True,
+  'page_changed_recently': True,
+  'context_page_description': 'Oxydactylus is an extinct genus of camelid endemic to North America. It lived from the Late Oligocene to the Middle Miocene, existing for approximately 14 million years. The name is from the Ancient Greek οξύς and δάκτυλος.\nThey had very long legs and necks, and were probably adapted to eating high vegetation, much like modern giraffes. Unlike modern camelids, they had hooves, rather than tough sole-pads, and splayed toes.',
+  'context_section_description': 'Oxydactylus is an extinct genus of camelid endemic to North America. It lived from the Late Oligocene to the Middle Miocene (28.4–13.7 mya), existing for approximately 14 million years. The name is from the Ancient Greek οξύς (oxys, "sharp")and δάκτυλος (daktylos, "finger").\n \nThey had very long legs and necks, and were probably adapted to eating high vegetation, much like modern giraffes. Unlike modern camelids, they had hooves, rather than tough sole-pads, and splayed toes.'
+}
+```
+### Data Fields
+
+- `language`
+- `page_url`
+- `image_url`
+- `page_title`
+- `section_title`
+- `hierarchical_section_title`
+- `caption_reference_description`
+- `caption_attribution_description`
+- `caption_alt_text_description`
+- `mime_type`
+- `original_height`
+- `original_width`
+- `is_main_image`
+- `attribution_passes_lang_id`
+- `page_changed_recently`
+- `context_page_description`
+- `context_section_description`
+
+Details on the field content can be found directly in the [paper, figure 5 and table 12.](https://arxiv.org/abs/2103.01913)
+
+### Data Splits
+
+All data is held in `train` split, with a total of 37046386 rows.
+
+## Dataset Creation
+
+### Curation Rationale
+
+[More Information Needed]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[More Information Needed]
+
+#### Who are the source language producers?
+
+[More Information Needed]
+
+### Annotations
+
+#### Annotation process
+
+[More Information Needed]
+
+#### Who are the annotators?
+
+[More Information Needed]
+
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[More Information Needed]
+
+### Discussion of Biases
+
+[More Information Needed]
```

Comment:
Maybe we can add this paragraph to this section (Section 3.4 in the paper):
```
Lastly we found that certain image-text pairs occurred very
frequently. These were often generic images that did not have
much to do with the main article page. Common examples
included flags, logos, maps, insignia and such. To prevent
biasing the data, we heavily under-sampled all such images
```